gguf-py/gguf/tensor_mapping.py

```diff
@@ -1193,36 +1165,27 @@ class TensorNameMap:
             "vision_tower.vision_model.embeddings.class_embedding",
             "model.vision_tower.embeddings.cls_token", # Intern-S1
             "vision_model.class_embedding", # llama 4
-            "model.vision.patch_embedding.cls_embedding", # cogvlm
         ),
 
         MODEL_TENSOR.V_ENC_EMBD_PATCH: (
             "vision_tower.vision_model.embeddings.patch_embedding",
             "model.vision_tower.embeddings.patch_embeddings.projection", # Intern-S1
             "vpm.embeddings.patch_embedding",
-            "model.vision_model.embeddings.patch_embedding", # SmolVLM
+            "model.vision_model.embeddings.patch_embedding", # SmolVLM, Janus Pro
```

Comment:
```suggestion
            "model.vision_model.embeddings.patch_embedding", # SmolVLM
```
No need to stack on for every model, the list will get very long that way. :)